# IMU bias msg.

std_msgs/Header header

geometry_msgs/Vector3 ang
geometry_msgs/Vector3 acc
geometry_msgs/Vector3 mag